test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/OpProvider.java

```diff
@@ -52,6 +52,7 @@
 			LIVE_HASH_NOT_FOUND,
 			INVALID_SIGNATURE,
 			INSUFFICIENT_PAYER_BALANCE,
+			UNKNOWN,
```

Comment:
Why is `UNKNOWN ` a permissible outcome?

Reply:
After client received the OK response code services after it sent a transaction to hedera services, the client knows its transaction passed the precheck. Now it will wait for the hashgraph's processing. If platform can't reach concensus in time, the test client don't know what happens to this transaction and it will put an UNKNOWN as the final status of this transaction. From code logic, this is a legitimate status to client. 

From the HAPI API document, I am not sure honestly. I guess we need to have a discussion with the team to decide what exactly all these response code means and under want conditions we should and shouldn't get them.

Reply:
Yes agreed. I was just concerned, if all status are valid, then when will the test fail because of INVALID status. Thank you!!

Reply:
Anything else should be considered as failure case. For example, if we try to update a file and get back a INVALID_FILE_ID, which should be considered validate. But if instead we get a INVALID_TOPIC_ID, then this is definitely a failure and indicate there is a severe issue on server side.